_data/people/matthewfeickert.yml

```diff
@@ -138,6 +143,7 @@ presentations:
     meetingurl: https://lpsc-indico.in2p3.fr/event/2585/
     location: Virtual
     focus-area: as
+    project:
```

Comment:
@henryiii are blank `project` fields okay if it doesn't really fit into any ongoing project specifically? Or is something else preferred.

Reply:
Blank is correct.